source/cpp/source/DefaultCommandHandler.cpp

```diff
@@ -424,6 +447,10 @@ std::optional<Response> DefaultCommandHandler::process (const Command & command)
             {"invoke-menu", [&] (auto && command) { return invokeMenu (command); }},
             {"get-slider-value", [&] (auto && command) { return getSliderValue (command); }},
             {"set-slider-value", [&] (auto && command) { return setSliderValue (command); }},
+            {
+                "set-text-editor-text",
+                [&] (auto && command) { return setTextEditorText (command); },
+            },
```

Comment:
V minor but I'm not sure this formatting is right. It looks like the string literal should be on the same line as the opening brace to be consistent with the arguments.

Reply:
Yeah it does look a little weird. It must go over the line break threshold for clang-format. I could turn it off for this section but I think I'd rather just let clang-format decide.